src/main/kotlin/no/nav/klage/oppgave/clients/OppgaveClient.kt

```diff
@@ -12,17 +12,15 @@ import org.springframework.web.reactive.function.client.bodyToMono
 class OppgaveClient(
     private val oppgaveWebClient: WebClient,
     private val stsClient: StsClient,
-    private val tracer: Tracer
+    private val tracer: Tracer,
+    @Value("\${spring.application.name}") val applicationName: String
```

Comment:
Hva er grunnen til å flytte denne hit? Er det et mønster vi bør bruke i andre apper også?

Reply:
Constructor injection er å foretrekke av flere grunner (https://stackoverflow.com/questions/21218868/explain-why-constructor-inject-is-better-than-other-options).

I dette tilfellet gjorde det også mye lettere å lage et objekt jeg kunne teste :)

Reply:
Skjønner! Skal kombinere det med litt bredere testing fremover.